Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs

```diff
@@ -12,6 +13,10 @@ public class DomainProfile
     {
         //[anything]@[anything].[anything]
         private string _email;
+
+        public List<string> FollowerEmails;
+        public List<string> FollowingEmails;
+
```

Comment:
public fields are bad, address code smells

Reply:
having full properties here isn't really needed in this PR. you could have left it as auto-properties, or as properties that throw NotImplementedException (like the IDE lightbulb will generate for you if you try to reference them while they don't exist at all). but it's good enough.

also, when you make changes to address previous review comments, click "resolve conversation" on them, after leaving a reply indicating how you addressed it if it's not obvious.